### PR TITLE
Kh/patch5

### PR DESCRIPTION
**Pull request by KellieHawks**
BD-273: taking Timing based functionalities confluence page, putting into docs
BD-385: clarification on in-app message HTML directions
BD-398: clarifying email address testing segment filter

from kh/wiki_metadata_patch
BD-416: Conversion Events Clarification in Glossary
BD-399: Specify that Push Open Events in currents refer only to "Direct Opens"
BD-394: Remove references to custom endpoints
Updating @EmilyNecciai -> @KellieHawks
# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)... because...

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>
- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Comment by KellieHawks** [comment]
@EmilyNecciai somewhat confused, it seems as though the two branches I was working on and finished merged into this larger branch? combining kh/wiki_metadata_add and kh/patch5 into kh/patch5.

**Comment by KellieHawks** [comment]
> @EmilyNecciai somewhat confused, it seems as though the two branches I was working on and finished merged into this larger branch? combining kh/wiki_metadata_add and kh/patch5 into kh/patch5.

I believe because they both merged, we can close the original kh/wiki_metadata_add branch. Just double-checking before I do anything serious.

File: _docs/_user_guide/engagement_tools/campaigns/scheduling_and_organizing/time_based_campaign.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+page_type: reference
+description: "This reference article covers time-based functionalities for Campaigns."
+---
+# Time-Based Functionalities for Campaigns
+
+> This document serves to inform you of the nuances of any time-related capabilities in Braze relating to campaigns to assist with customer questions/strategies, as well as with troubleshooting. It outlines the definition of that term or capability, the time zone it pertains to if any, an example for clarity, and other additional caveats and notes.
+
+Scheduled Delivery| Definition | Time Zone | Example | Notes |
+----------------- | ---------- | --------- | ------- | ----- |
+Send at a designated time (except for Local Time Delivery) | Calendar Day | Company's Time Zone | | |
```
Comment (EmilyNecciai) [inline _docs/_user_guide/engagement_tools/campaigns/scheduling_and_organizing/time_based_campaign.md]: @KellieHawks Are there meant to be examples in here?

Comment (KellieHawks) [reply _docs/_user_guide/engagement_tools/campaigns/scheduling_and_organizing/time_based_campaign.md]: Yes and No, I was just trying to move over all the information on this from confluence. 
Some lines have examples, some did not. 

https://confluence.braze.com/pages/viewpage.action?spaceKey=WS&title=Dictionary+of+Timing-Based+Functionalities

Comment (KellieHawks) [reply _docs/_user_guide/engagement_tools/campaigns/scheduling_and_organizing/time_based_campaign.md]: @EmilyNecciai

**Pull request merged**